bioimageio_chatbot/chatbot.py

```diff
@@ -199,7 +217,10 @@ async def register_chat_service(server):
     description_by_id = {collection['id']: collection['description'] for collection in collections}
     customer_service = create_customer_service(knowledge_base_path)
 
-    async def chat(text, chat_history, user_profile=None, channel=None, status_callback=None, context=None):
+    async def report(user_report, context=None):
+        print(f"UserFeedback: {user_report}")
```

Comment:
We need to call the save chat history here too. The difference is that it should contain additional key "comment" and "like", and save in a different folder.